<launch>

  <node name="fps_control" pkg="ros_visualise" type="fps_control.py" output="screen" >
    <param name="topic_sub" value="/image_raw" />
    <param name="topic_pub" value="/controled_stream" />
    <param name="fps" value="5" />
  </node>

</launch>
